lib/src/sql/value/serde/ser/value/mod.rs

```diff
@@ -807,7 +807,7 @@ mod tests {
 
 	#[test]
 	fn function() {
-		let function = Box::new(Function::Cast(Default::default(), Default::default()));
+		let function = Box::new(Function::Normal(Default::default(), Default::default()));
```

Comment:
Due to the two `Default::default()`'s, it's quite unclear what this test was testing (both before and after the change)